client/yarn.lock

```diff
@@ -925,6 +925,11 @@ duplexify@^3.4.2, duplexify@^3.6.0:
   resolved "https://registry.yarnpkg.com/elkjs/-/elkjs-0.8.2.tgz#c37763c5a3e24e042e318455e0147c912a7c248e"
   integrity sha512-L6uRgvZTH+4OF5NE/MBbzQx/WYpru1xCBE9respNj6qznEewGUIfhzmm7horWWxbNO2M0WckQypGctR8lH79xQ==
 
+"elkjs-9-1@npm:elkjs@0.9.1":
```

Comment:
I think elkjs latest should also point to 0.9.1 instead of 0.7.1 in the yarn.lock.